site/_layouts/posts.html

```diff
@@ -25,7 +25,11 @@ <h1>{{ page.title }}</h1>
                 {% endif %}
 
                 <div class="post-single-meta-author-name">
-                  <a href="/tags/{{ page.author_name | urlencode }}">{{ page.author_name }}</a>
+                    {% if page.author_url %}
```

Comment:
I guess I pretty strongly prefer this version:
 - we have only a handful of articles so 'other articles written by author' so the list is pretty minimal to sift through and wouldn't provide better exposure for the blog or the author IMO
 - It seems like if they want to know about that author, it would make sense to link to the place the author wants (github, twitter, etc) and not just a subset of their work (only the sonobuoy.io blogs by that person)

I just dont really see the use case for the subset of blog articles being the best link, especially if set on the page.

If you don't want me linking to my github then I can just remove the `author_url` and leave this snippet the same so that it's functionality will work intuitively in the future.

Reply:
All valid concerns, thank you!
Let's get this merged.